src/Goerli-DssSpell.sol

```diff
@@ -89,58 +50,98 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
-        // Run sanity checks
-        require(TeleportFeeLike(LINEAR_FEE_STA).fee() == WAD / 10000);
-        require(TeleportFeeLike(LINEAR_FEE_STA).ttl() == 30 minutes); // finalization time on Goerli
-        require(StarknetTeleportBridgeLike(TELEPORT_GATEWAY_STA).l1Escrow() == ESCROW_STA);
-        require(StarknetTeleportBridgeLike(TELEPORT_GATEWAY_STA).l1TeleportRouter() == ROUTER);
-        require(StarknetTeleportBridgeLike(TELEPORT_GATEWAY_STA).l1Token() == DAI);
-        require(StarknetTeleportBridgeLike(TELEPORT_GATEWAY_STA).l2TeleportGateway() == TELEPORT_L2_GATEWAY_STA);
-        require(StarknetTeleportBridgeLike(TELEPORT_GATEWAY_STA).starkNet() == STARKNET_CORE);
+    uint256 constant WAD            = 10 ** 18;
+    uint256 constant MILLION        = 10 ** 6;
 
-        // Increase system debt ceilings
-        DssExecLib.increaseIlkDebtCeiling(ILK, CEILING, true);
+    uint256 constant FIFTY_PCT_RATE = 1000000012857214317438491659;
 
-        // Configure TeleportJoin
-        TeleportJoinLike(TELEPORT_JOIN).file("fees", DOMAIN_STA, LINEAR_FEE_STA);
-        TeleportJoinLike(TELEPORT_JOIN).file("line", DOMAIN_STA, CEILING * WAD);
+    address immutable MIP21_LIQUIDATION_ORACLE = DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
 
-        // Configure TeleportRouter
-        TeleportRouterLike(ROUTER).file("gateway", DOMAIN_STA, TELEPORT_GATEWAY_STA);
+    function _updateDoc(bytes32 ilk, string memory doc) internal {
+        ( , address pip, uint48 tau, ) = RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).ilks(ilk);
+        require(pip != address(0), "DssSpell/unexisting-rwa-ilk");
 
-        // Authorize TeleportGateway to use the escrow
-        EscrowLike(ESCROW_STA).approve(DAI, TELEPORT_GATEWAY_STA, type(uint256).max);
-
-        // Configure Chainlog
-        DssExecLib.setChangelogAddress("STARKNET_TELEPORT_BRIDGE", TELEPORT_GATEWAY_STA);
-        DssExecLib.setChangelogAddress("STARKNET_TELEPORT_FEE", LINEAR_FEE_STA);
-
-        DssExecLib.setChangelogVersion("1.14.4");
-
-        // ------------------ MOMC Parameter Changes -----------------
-        // https://vote.makerdao.com/polling/QmahDuNx#poll-detail
-
-        // Increase the MANA-A Stability Fee from 4.5% to 7.5%
-        DssExecLib.setIlkStabilityFee("MANA-A", SEVEN_PT_FIVE_PERCENT_RATE, true);
+        // Init the RwaLiquidationOracle to reset the doc
+        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).init(
+            ilk, // ilk to update
+            0,   // price ignored if init() has already been called
+            doc, // new legal document
+            tau  // old tau value
+        );
+    }
 
-        // Decrease the MANA-A line from 17 million DAI to 10 million DAI
-        DssExecLib.setIlkAutoLineDebtCeiling("MANA-A", 10 * MILLION);
+    function actions() public override {
+        // -------------------- Update RWA007 Legal Documents ---------------------
+        // https://forum.makerdao.com/t/nov-16-2022-executive-contents/18747
```

Comment:
the link same the same across the changes, consider grouping them together under a general `Update RWAs Legal Documents` comment

Reply:
done